oecophylla/cli/tests/data/tool_params.yml

```diff
@@ -26,15 +26,15 @@ metaphlan2:
   # taxon name to TaxID dictionary
   name2tid: 
   levels: phylum,genus,species
-  map: no
+  map: yes
```

Comment:
Why changing these defaults?

Reply:
I think that the test would better cover all possible functions. So I turned on the "keep mapping file" function for all.

Reply:
hmm, this is a good point. I'd also been using these effectively as defaults, but this reminds me that I need a better parameter explainer... 

Maybe we should keep these as 'no' for now?

Reply:
Sure! Will do.